src/components/auth/login/SignUpScreen.js

```diff
@@ -105,6 +105,7 @@ const SignupScreen = ({ screenProps, styles, store, handleLoginMethod, sdkInitia
               disabled={!sdkInitialized}
               testID="login_with_facebook"
               icon={facebookBtnIcon}
+              iconProps={{ viewBox: '0 0 11 22' }}
```

Comment:
```suggestion
              iconStyle={styles.signupIcon}
```

Reply:
As above, these are svg props, not styles